src/matrix.jl

```diff
@@ -41,6 +41,11 @@ getindex(M::IntervalMatrix, i::Int) = getindex(M.mat, i)
 setindex!(M::IntervalMatrix, X, inds...) = setindex!(M.mat, X, inds...)
 copy(M::IntervalMatrix) = IntervalMatrix(copy(M.mat))
 
+# constructor from uniform scaling
+function IntervalMatrix(αI::UniformScaling{Interval{T}}, m, n=m) where {T}
```

Comment:
```suggestion
function IntervalMatrix(αI::UniformScaling{Interval{T}}, m::Integer, n::Integer=m) where {T}
```
This avoids confusion when calling the constructor with unexpected arguments.